src/modules/dashboard/components/ExpendituresDialog/ExpendituresDialog.tsx

```diff
@@ -73,26 +73,23 @@ const ExpendituresDialog = ({
   prevStep,
   colony: { colonyAddress },
   colony,
+  isVotingExtensionEnabled,
   nextStep,
 }: Props) => {
   const { walletAddress, username, ethereal } = useLoggedInUser();
+  const { isOneTxPaymentExtensionEnabled } = useEnabledExtensions({
+    colonyAddress,
+  });
 
   const allUserRoles = useTransformer(getAllUserRoles, [colony, walletAddress]);
 
   const hasRegisteredProfile = !!username && !ethereal;
   const canCreatePayment =
     hasRegisteredProfile &&
-    canAdminister(allUserRoles) &&
-    canFund(allUserRoles);
-
-  const { data: colonyExtensionsData } = useColonyExtensionsQuery({
-    variables: { address: colonyAddress },
-  });
+    ((canAdminister(allUserRoles) && canFund(allUserRoles)) ||
+      isVotingExtensionEnabled);
 
-  const canMakePayment =
-    colonyExtensionsData?.processedColony?.installedExtensions?.find(
-      ({ extensionId }) => extensionId === Extension.OneTxPayment,
-    ) || false;
+  const canMakePayment = isOneTxPaymentExtensionEnabled || false;
```

Comment:
Hmmm..... I don't think this is correct

Either that `false` was added for debugging purposes, or it doesn't have a place in there

Reply:
it was there before but I have removed it now.